script/build.ts

```diff
@@ -55,15 +58,31 @@ updateLicenseDump(err => {
       process.exit(1)
     } else {
       console.log(`Built to ${appPaths}`)
-      process.exit(0)
     }
   })
 })
 
-function packageApp(callback) {
-  const options = {
-    name: distInfo.getExecutableName(),
-    platform: process.platform,
+function packageApp(
+  callback: (error: Error | null, appPaths: string | string[]) => void
+) {
+  // not sure if this is needed anywhere, so I'm just going to inline it here
+  // for now and see what the future brings...
+  const toPackagePlatform = (platform: NodeJS.Platform) => {
+    if (
+      process.platform === 'win32' ||
+      process.platform === 'darwin' ||
+      process.platform === 'linux'
+    ) {
+      return process.platform
+    }
+    throw new Error(
+      `Unable to convert to platform for electron-packager: '${process.platform}`
+    )
+  }
+
+  const options: packager.Options = {
+    name: getExecutableName(),
+    platform: toPackagePlatform(process.platform),
```

Comment:
This can just as easily be `toPackagePlatform()` if you prefer the previous function to call `process.platform` internally. IMO, if platform will always come from the process, may as well remove the argument from the function.